src/Halogen/Aff/Driver.purs

```diff
@@ -92,10 +93,10 @@ import Halogen.Query.Input (Input(..))
 -- | `const (pure unit)`.
 type RenderSpec h r =
   { render
-      :: forall s act ps o
+      :: forall s act ps o m
```

Comment:
I had a go at this, and I tried adding the m here like this as well, but I think that's the problem. Rendering does not require it to be fixed to `Aff`, however everything in `VDom.Driver` uses `Aff` in type signatures. The `m` parameter needs to be carried around in the types of everything, rather than foralled, I think.